src/reporting-config/reporting-config-slice.ts

```diff
@@ -81,3 +91,96 @@ export function selectReportingConfigLoadStatus( state: RootState ) {
 export function selectReportingConfigError( state: RootState ) {
 	return state.reportingConfig.error;
 }
+
+export function selectRelevantTaskIds( state: RootState ): string[] {
```

Comment:
Another core change: this lets us use the redux state tree to calculate which tasks we should show the user.